### PR TITLE
Update commander.py

### PR DESCRIPTION
**Pull request by Mis1ify**
Fix python commander.py errors

### CODE REVIEW

File: utils/commander.py [hunk 1]
```diff
@@ -51,14 +51,14 @@ def pathCompleter(text, state):
             readline.parse_and_bind("tab: complete")
             readline.set_completer(pathCompleter)
 
-            input_path = input(colored(f"Path of the image or folder ({colored(default_path, "yellow")}{f"{colored("): ", "blue")}"}", "blue")).strip()
+            input_path = input(colored(f"Path of the image or folder ({colored(default_path, 'yellow')}): ", "blue")).strip()
```
Comment (Tetrax-10) [inline utils/commander.py:54]:
```suggestion
            input_path = input(colored(f"Path of the image or folder ({colored(default_path, 'yellow')}{colored('): ', 'blue')}", "blue")).strip()
```

File: utils/commander.py [hunk 2]
```diff
@@ -51,14 +51,14 @@ def pathCompleter(text, state):
...
             args.path = glob.get_abs_path(glob.correct_path(input_path)) if input_path != "" else default_path
 
             if glob.is_file(args.path):
                 default_out = glob.join_path(glob.get_dirname(args.path), "cover-arts")
             else:
                 default_out = glob.join_path(args.path, "cover-arts")
-            input_out = input(colored(f"Output folder ({colored(default_out, "yellow")}{f"{colored("): ", "blue")}"}", "blue")).strip()
+            input_out = input(colored(f"Output folder ({colored(default_out, 'yellow')}): ", "blue")).strip()
```
Comment (Tetrax-10) [inline utils/commander.py:61]:
```suggestion
            input_out = input(colored(f"Output folder ({colored(default_out, 'yellow')}{colored('): ', 'blue')}", "blue")).strip()
```

File: utils/commander.py (hunk 1 above)
Comment (Tetrax-10) [reply utils/commander.py:54]: This is the output of your code:

image

as you can see the `):` is white, it should be blue.

File: README.md [hunk 3]
```diff
@@ -1,146 +1,5 @@
...
-
-1. Help me to build/test executable for `linux` and `mac os`.
-2. Help me fix the `known bugs`.
-
-</br>
-
-### Support
-
-Like This Tool? Gimme Some ❤️ by Liking this Repository.
+(This fork fixes utils/commander.py)
```
Comment (Tetrax-10) [inline README.md:5]: Please do not modify the `README.md` file in your pull request. If merged, your changes will alter the original `README.md`. Kindly revert the `README.md` to its original content by copying the original `README.md` content and pasting it into your repository's `README.md`. Then commit the changes.

**Review by Tetrax-10 — CHANGES_REQUESTED** [review]
Thank you for the PR, it's greatly appreciated. Please review the requested changes.

File: README.md (hunk 3 above)
Comment (Mis1ify) [reply README.md]: Done! (I think, it's my first time using this feature) Sorry for problems.

**Comment by Mis1ify** [comment]
I reviewed the changes and reverted `README.md`.

**Review by Tetrax-10 — APPROVED** [review]

**Pull request merged**